mollie-payments-for-woocommerce/includes/mollie/wc/helper/settings.php

```diff
@@ -14,114 +47,62 @@ public function isTestModeEnabled ()
      * @param bool $test_mode
      * @return null|string
      */
-    public function getApiKey ($test_mode = false)
+    public function getApiKey($test_mode = false)
     {
         $setting_id = $test_mode ? 'test_api_key' : 'live_api_key';
 
         return trim(get_option($this->getSettingId($setting_id)));
     }
 
-	/**
-	 * Order status for cancelled payments
-	 *
-	 * @return string|null
-	 */
-	public function getOrderStatusCancelledPayments ()
-	{
-		return trim(get_option($this->getSettingId('order_status_cancelled_payments')));
-	}
-
     /**
-     * @return string
+     * Order status for cancelled payments
+     *
+     * @return string|null
      */
-    protected function getPaymentLocaleSetting ()
+    public function getOrderStatusCancelledPayments()
```

Comment:
Looks dangerous, but I assume that you have made sure that this method is not called from outside.

Reply:
Technically, this is an API change.